casr/tests/tests.rs

```diff
@@ -5841,14 +5843,8 @@ fn test_casr_afl_csharp() {
 
     let _ = fs::remove_dir_all(&paths[1]);
     let _ = fs::create_dir(abs_path("tests/tmp_tests_casr"));
-    let _ = Command::new("cp")
-        .args(["-r", &paths[2], &paths[4]])
-        .output()
-        .expect("failed to copy dir");
-    let _ = Command::new("cp")
-        .args(["-r", &paths[3], &paths[5]])
-        .output()
-        .expect("failed to copy dir");
+    let _ = copy_dir(&paths[2], &paths[4]);
```

Comment:
Can you do the same in other similar places (with cp -r) in this file?

Reply:
done